openedx_lti_tool_plugin/resource_link_launch/exceptions.py

```diff
@@ -1,4 +1,4 @@
-"""Exceptions for openedx_lti_tool_plugin."""
+"""Exceptions."""
```

Comment:
@anfbermudezme I think is redundant and also these exceptions aren't for the whole `openedx_lti_tool_plugin` module but rather the `resource_link_launch` module, also if we had to change the `openedx_lti_tool_plugin` module in the future it will be easier to have unnecessary references to such name like in this docstring.